client/gutenberg/extensions/tiled-gallery/variables.scss

```diff
@@ -4,5 +4,5 @@ $tiled-gallery-add-item-border-color: #555d66; // Gutenberg $dark-gray-500
 $tiled-gallery-add-item-border-width: 1px; // Gutenberg $border-width
 $tiled-gallery-caption-background-color: #000;
 $tiled-gallery-content-width: 610px; // Gutenberg $content-width
-$tiled-gallery-margin: 2px; // Fixed in JS, see `LayoutStyles` from `edit.jsx`
+$tiled-gallery-gutter: 4px; // Fixed in JS, see `LayoutStyles` from `edit.jsx`
```

Comment:
Let's loop back later to rename margin to gutter also elsewhere to keep things consistent: 

https://github.com/Automattic/wp-calypso/blob/f76c1442bb5ef5812d56cd7516bb6f1c94de4f52/client/gutenberg/extensions/tiled-gallery/constants.js#L34